### DISCUSSION TITLE
preprocessing table output description

### DISCUSSION THREAD

**Discussion by singhd6** [milaboratory/mixcr#1875] [Q&A] [answered]
i ran the follwoing command in mixcr for v gene usage analysis, my reads are without umi tag
![image](https://github.com/user-attachments/assets/4afd4194-ea3b-4abd-abf2-506c63d290b3)
i want to understand the preprocessing step 
![image](https://github.com/user-attachments/assets/a1ef8272-536c-4e66-8ccb-44e900435681)
specifically the preprocessing step: 
what does <html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/singhd6/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/singhd6/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#467886" vlink="#96607D">

preprocessor
--
Filter [IGH] \| Filter stops in CDR3, OOF   in CDR3

</body>

</html>
my BCR sequencing data is IgH data, then why is there IgH filter <html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/singhd6/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/singhd6/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#467886" vlink="#96607D">

preprocessor
--
Filter [IGH] \| Filter stops in CDR3, OOF   in CDR3

</body>

</html>

**Comment by mizraelson** [comment, marked as answer]
If there are multiple chains in the data the analysis will be performed separately for each chain. IGH simply means that this is the number for all IGH clonotypes.
